scripts/lib/install

```diff
@@ -212,17 +212,16 @@ if [ -f /etc/os-release ]; then
 fi
 
 case "$os_id $os_version_id" in
-    'debian 10' | 'debian 11' | 'ubuntu 20.04') ;;
+    'debian 11' | 'ubuntu 20.04') ;;
     *)
         set +x
         cat <<EOF
 
 Unsupported OS release: $os_id $os_version_id
 
 Zulip in production is supported only on:
```

Comment:
This would be an orthogonal change, but what do you think about changing this string to say "This Zulip version supports the following host operating systems:"?  @alexmv FYI as well.